webservices/resources/filings.py

```diff
@@ -76,3 +76,40 @@ class FilingsList(BaseFilings):
     @property
     def args(self):
         return utils.extend(super().args, args.entities)
+
+
+@doc(
+    tags=['efiling'],
+    description=docs.EFILE_FILES,
+)
+class EFilingsView(views.ApiResource):
+
+    model = models.EFilings
+    schema = schemas.EFilingsSchema
+    page_schema = schemas.EFilingsPageSchema
+
+    filter_multi_fields = [
+        ('file_number', models.Filings.file_number),
+        ('committee_id', models.Filings.committee_id),
+    ]
+
+    @property
+    def args(self):
+        return utils.extend(
+            args.paging,
+            args.efilings,
+            args.make_sort_args(
+                default='-receipt_date',
+                # TODO: add indexes and turn on validation
+                # validator=args.IndexValidator(models.EFilings),
```

Comment:
Not a problem here, just a note and question:  I took a look at the `IndexValidator` more closely as I hadn't seen it before, and it makes me wonder if we should have this on the views for the itemized schedules as well.  It's not currently there and if we ever run into a situation again where the indexes get dropped or aren't set properly, it appears this would also help alleviate the problem, right?